### ISSUE TITLE
List, stats, and review page do not load correctly after editing project (bad gateway error)

### ISSUE THREAD

**Issue by justinjjvanderhooft** [iomega/paired-data-form#188]
**Describe the bug**
List, stats, and review page do not load correctly after editing project and provides a bad gateway error

**To Reproduce**
1. Go to https://pairedomicsdata.bioinformatics.nl/projects and wait a bit.

**Expected behavior**
Normally the projects are loaded in the list.

**Desktop (please complete the following information):**
 - OS:  iOS
 - Browser: safari

**Additional context**
This happened after updating one of the projects and approving the project under review.

**Comment by sverhoeven** [comment]
Seems the redis and api service where down.
I restarted both of them and pages work again.

On further investigation, seems redis died to memory
```
redis_1   | PANIC: unprotected error in call to Lua API (not enough memory)
```

The api seems to have died when it failed to publish to zenodo
```
failedReason: 'Zenodo communication error determining latest version from concept: NOT FOUND',
```
I manually published to Zenodo with `docker-compose exec api npm run publish2zenodo` without problems, so probably Internet was having bad day when this error was occured.
